template-transformer/handler.lua

```diff
@@ -72,6 +72,7 @@ function TemplateTransformerHandler:access(config)
     local transformed_body = template_transformer.get_template(config.request_template){query_string = query_string,
                                                                                         headers = headers,
                                                                                         body = body,
+                                                                                        custom_data = ngx.ctx.custom_data,
```

Comment:
Why did we choose not to inject in the headers and instead use a custom_data?

Reply:
There's no "default" header for this, and using the ngx context we can add future features with this more easily.